.frigg.yml

```diff
@@ -3,4 +3,3 @@ setup_tasks:
 
 tasks:
  - npm test
- - npm run lint
```

Comment:
Oh, didn't realise that earlier. Can you please keep them separate. They were separate for a reason. It is way easier to see what is failing when they are separate tasks.